parser-core/src/main/parse/StructureParser.scala

```diff
@@ -138,18 +138,19 @@ object StructureParser {
 
     splitOnProcedureStarts(tokens, Seq()).flatMap(procedureSyntax).toMap
   }
-
+  @throws(classOf[CompilerException])
   def findIncludes(tokens: Iterator[Token]): Seq[String] = {
     val includesPositionedTokens =
       tokens.dropWhile(! _.text.equalsIgnoreCase("__includes"))
     if (includesPositionedTokens.isEmpty)
       Seq()
     else {
       includesPositionedTokens.next
-      if (includesPositionedTokens.next.tpe != core.TokenType.OpenBracket)
-        Seq()
+      val includesWithoutComments = includesPositionedTokens.filter(_.tpe != core.TokenType.Comment)
+      if (includesWithoutComments.next.tpe != core.TokenType.OpenBracket)
+        exception("Missing open bracket in structureParser", tokens.next)
```

Comment:
Should this be something like "Did not find expected open bracket for __includes declaration"?  We'll know which file it's in from the stack trace, so it should be something that might help out a user who managers to trigger the error.

Reply:
Yea, I'll update the exception to better reflect the error.